api/src/main/java/org/openmrs/module/fhir2/api/dao/FhirConceptDao.java

```diff
@@ -13,7 +13,7 @@
 
 import org.openmrs.Concept;
 
-public interface FhirConceptDao {
+public interface FhirConceptDao extends FhirDao<Concept> {
 	
 	Optional<Concept> getConceptByUuid(String uuid);
```

Comment:
Change this from  getConceptByUuid(String uuid);  to get(String uuid);, You could also Check FhirConceptService.java